### ISSUE TITLE
Using a pointcloud with pg.meshtools.createMesh 

### ISSUE THREAD

**Issue by Coastal0** [gimli-org/gimli#98]
Hi again...

Trying to make a grid from a point cloud (I'm still stuck on that apparently..).

I saw that createMesh can take a list, but I can't seem to work it out.

How can I generate a mesh from a list of points?

> List of x y pairs [[x0, y0], ... ,[xN, yN]]

```
coords = np.random.rand(10,2)*10
mesh = pg.meshtools.createMesh(coords,quality=33)

```

```
Traceback (most recent call last):

  File "<ipython-input-33-41f51f2d9570>", line 1, in <module>
    Mesh_FePoly = pg.meshtools.createMesh(coords,quality=33)

  File "F:\WinPython-64bit-3.6.2.0Qt5\python-3.6.2.amd64\lib\site-packages\pygimli\meshtools\mesh.py", line 73, in createMesh
    if poly.dim() == 2:

AttributeError: 'numpy.ndarray' object has no attribute 'dim'
```

https://www.pygimli.org/pygimliapi/_generated/pygimli.meshtools.html#pygimli.meshtools.createMesh

**Comment by Coastal0** [comment]
I was able to solve the problem using
`np.ndarray.tolist(coords)`

However, I seem to be irregularly crashing python when using pg.show(mesh) on my full solution of ~8500 nodes.

**Comment by Coastal0** [comment]
Is it further possible to generate the mesh from the point-cloud, while including a boundary polygon?

For example, my mesh from the point-cloud returns the figure below; however it has meshed across the topography.

Can I include both a polygon and list of points into pg.meshtools.createMesh?

![figure_1-4](https://user-images.githubusercontent.com/25240257/31974082-089dd300-b95c-11e7-947b-040d943a257c.png)

**Comment by halbmy** [comment]
If you provide a point cloud, the convex hull is used as boundary. Of course you can use your own boundary. Create an empty plc/mesh, add the nodes one by one using `plc.createNode` and add your boundary using `plc.createEdge`. In the meshtools API and the examples you will see how to do it.

**Comment by Coastal0** [comment]
Thankyou.

If I've done it correctly, I should be able to see them with `pg.show(myPLC)` right?

**Comment by halbmy** [comment]
Yes, just plot the plc with `pg.show`. 
You can just plot your points onto the mesh and then you see whether they all coincide with nodes.

**Comment by carsten-forty2** [comment]
should be solved in #101
